src/components/vm/consoles/VncConsole.tsx

```diff
@@ -126,12 +155,11 @@ export const VncConsole = ({
     ]);
 
     React.useEffect(() => {
-        initLogging(vncLogging);
         try {
             connect();
             onConnected(novncElem.current);
         } catch (e) {
-            onInitFailed && onInitFailed(e);
+            if (onInitFailed) onInitFailed(e);
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2267-42a6eadf-20250808-101729-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)